client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -139,8 +142,8 @@ export class EditorMediaModalDetailItem extends Component {
 		const { item, translate } = this.props;
 
 		//do a simple guid vs url check
-		const guidParts = url.parse( item.guid );
-		const URLParts = url.parse( item.URL );
+		const guidParts = getUrlParts( item.guid );
+		const URLParts = getUrlParts( item.URL );
```

Comment:
Just curious to know why the decision of moving from `url` package and use `getUrlParts` from `@automattic/calypso-url` was made?

Reply:
@yansern The linter brought me there. The `url` package is deprecated.